skore/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Please remove this file as it's generated, i will add a new line in the .gitignore.